index.html

```diff
@@ -4,9 +4,152 @@
+	<link rel="stylesheet" href="assets/css/home.css">
 	<title>Javascript Hackathon - Home page</title>
 </head>
 <body>
+  <header >
+		<nav class="navbar hide" id="mobile-menu">
+		  <div>
+        <img src="assets/images/icons/Cancel-Icon.svg" alt="menu icon" class="menu-icon" id="cancle-icon">
+			</div>
+		  <ul class="nav-items">
+				<li class="nav-item"><a href="about.html">About</a></li>
+				<li class="nav-item"><a href="">Program</a></li>
+				<li class="nav-item"><a href="">Join</a></li>
+				<li class="nav-item"><a href="">Sponser</a></li>
+				<li class="nav-item"><a href="">News</a></li>
+				<li class="nav-item nav-btn"><a href="">CC Campaign</a></li>
+			</ul>
+		</nav>
+	</header>
+	<header class="desktop">
+    <article class="toolbar">
+      <a href="#"><img src="assets/images/icons/facebook-icon.png" alt="facebook icon" class="toolbar-item"></a>
+			<a href="#"><img src="assets/images/icons/twitter.png" alt="twitter icon" class="toolbar-item"></a>
+			<a href="#" class="toolbar-item">English</a>
+			<a href="#" class="toolbar-item">My page</a>
+			<a href="#" class="toolbar-item">Logout</a>
+		</article>
+    <nav class="navbar-desktop">
+			<a href="index.html"><img src="assets/images/logo.png" alt="logo" class="nav-logo"></a>
+			<ul class="nav-items-desktop">
+        <li><a href="about.html"class="nav-item-desktop">About</a></li>
+				<li><a href="#" class="nav-item-desktop">Program</a></li>
+				<li><a href="#" class="nav-item-desktop">Join</a></li>
+				<li><a href="#" class="nav-item-desktop">Sponser</a></li>
+				<li><a href="#" class="nav-item-desktop">News</a></li>
+				<li class="nav-item-desktop nav-btn">
+           <button class="top-btn">JS Hackathon</button>
+				</li>
 
+			</ul>
+		</nav>
+	</header>
+  <div class="container show" id="main-body">
+		<header class="mobile">
+			<nav class="navbar">
+				<div>
+					<img src="assets/images/icons/hamburger-icon.svg" alt="menu icon" class="menu-icon" id="menu-icon"></div>
+			</nav>
+		</header>
+		<section class="hero">
+		  <div class="hero-heading">
+				<span><p class="supporting-text">"Hello coding world"</p></span>
+				<h1 class="primary-text">Javascript Devs Hackathon 2023</h1>
+			</div>
+			<div>
+				<p class="description-hero">
+					Javascript Hackathon is an annual weekend long beginer friendly hackathon series for Javascript enthusiasts with developers from all over the world and different skill levels.
+				</p>
+			</div>
+			<div class="date-venue">
+				<h6 class="date">2023.02.04(SAT) ~ 5(SUN)</h6>
+				<p class="description venue">@ National Museum of Karea, Art Center Nabi and more
+        </p>
+			</div>
+		</section>
+		<section class="program">
+			<h2 class="main-title dark">Main Program</h2>
+			<hl class="divider"></hl>
+			<div class="program-cards-container">
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">Lecture</h3>
```

Comment:
- Kindly make sure you match this original design, update your main hero text to look like this below, and check the difference, please.
- Kindly make your background match the original design, update your home page to look like this below 

Your design
![image](https://user-images.githubusercontent.com/60612329/212443148-b71ce03e-7b72-4af0-a83a-0d8691e34824.png)

required design
![image](https://user-images.githubusercontent.com/60612329/212443167-acd66e1f-2051-4ccb-ac79-6e69ffe1d632.png)

Reply:
Thank you for your feedback there was issue with the relative path I used for the images were not loading properly, I have now fixed the issue.